kodilla-advanced-tests/src/test/java/com/kodilla/mockito/homework/WeatherAlertServiceTestSuite.java

```diff
@@ -0,0 +1,118 @@
+package com.kodilla.mockito.homework;
+
+import com.kodilla.mockito.Notification;
+import org.junit.jupiter.api.*;
+import org.mockito.Mockito;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WeatherAlertServiceTestSuite {
+
+    WeatherAlertService weatherAlertService = new WeatherAlertService();
+    Location location1 = Mockito.mock(Location.class);
```

Comment:
Takie rzeczy w metodzie before

Reply:
Z Before próbowałem jak przy ostatnim zadaniu, ale nie idzie. Czy o to chodzi i probować dalej ?
Powinienem do Before normanie dodać po kolei:
location1
location2
client1
client2
...   
?

Reply:
Chodzi o to, aby ten caly serwis inicjowac dla kazdej metody w metodzie init. Zeby kazda metida miala swiezy serwis.